thrifty-java-codegen/src/main/kotlin/com/microsoft.thrifty.gen/GenerateReaderVisitor.kt

```diff
@@ -51,7 +51,8 @@ internal open class GenerateReaderVisitor(
         private val resolver: TypeResolver,
         private val read: MethodSpec.Builder,
         private val fieldName: String,
-        private val fieldType: ThriftType
+        private val fieldType: ThriftType,
+        private val acceptUnknownEnumValues: Boolean = false
```

Comment:
I'd prefer if, for this class, we called it `failOnUnknownEnumValues` - that's a clearer indication on what this class will do, in my opinion.

Reply:
Will change it. Would you prefer the command line flag also be changed?